### PR TITLE
Docs: Renderer token

### PR DESCRIPTION
**Pull request by xlson**
**What is this feature?**

Documentation for the renderer auth token

**Why do we need this feature?**

**Who is this feature for?**

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by xlson** [comment]
Not sure about backporting, feels like we might need to but I'm not sure if we should do every version or not. @joanlopez do you know how we have been treating these kinds of docs pr:s recently?

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 1]
```diff
@@ -1795,6 +1795,12 @@ This option does not require any configuration.
 
 Options to configure a remote HTTP image rendering service, e.g. using https://github.com/grafana/grafana-image-renderer.
 
+#### renderer_token
+
+> **Note**: Available in Grafana v9.1.2 and Image Renderer v3.6.1 or later versions.
+
+An auth token that will be sent to and verified by the renderer. The renderer will deny any request without an auth token matching the one configured on the renderer side.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-grafana/_index.md:1802]:
```suggestion
An auth token will be sent to and verified by the renderer. The renderer will deny any request without an auth token matching the one configured on the renderer.
```

File: docs/sources/setup-grafana/image-rendering/_index.md [hunk 2]
```diff
@@ -55,6 +55,33 @@ docker run -d --name=renderer --network=host -v /some/path/config.json:/usr/src/
 
 You can see a docker-compose example using a custom configuration file [here](https://github.com/grafana/grafana-image-renderer/tree/master/devenv/docker/custom-config).
 
+### Security
+
+> **Note:** This feature is available in Image Renderer v3.6.1 and later versions.
```
Comment (chri2547) [inline docs/sources/setup-grafana/image-rendering/_index.md:60]:
```suggestion
> **Note:** This feature is available in Image Renderer v3.6.1 and later.
```

File: docs/sources/setup-grafana/image-rendering/_index.md [hunk 3]
```diff
@@ -55,6 +55,33 @@ docker run -d --name=renderer --network=host -v /some/path/config.json:/usr/src/
...
+
+Restrict access to the rendering endpoint by specifying a secret token. The token should be configured in both the Grafana configuration file and the renderer configuration file. This token becomes especially important when running the plugin in remote rendering mode.
```
Comment (chri2547) [inline docs/sources/setup-grafana/image-rendering/_index.md:62]:
```suggestion
You can restrict access to the rendering endpoint by specifying a secret token. The token should be configured in the Grafana configuration file and the renderer configuration file. This token is important when you run the plugin in remote rendering mode. 
```

File: docs/sources/setup-grafana/image-rendering/_index.md [hunk 4]
```diff
@@ -55,6 +55,33 @@ docker run -d --name=renderer --network=host -v /some/path/config.json:/usr/src/
 
 You can see a docker-compose example using a custom configuration file [here](https://github.com/grafana/grafana-image-renderer/tree/master/devenv/docker/custom-config).
 
+### Security
+
+> **Note:** This feature is available in Image Renderer v3.6.1 and later versions.
+
+Restrict access to the rendering endpoint by specifying a secret token. The token should be configured in both the Grafana configuration file and the renderer configuration file. This token becomes especially important when running the plugin in remote rendering mode. 
+
+Renderer version v3.6.1 or later will require a Grafana version where this feature is also available to run, these include:
```
Comment (chri2547) [inline docs/sources/setup-grafana/image-rendering/_index.md:64]:
```suggestion
Renderer version v3.6.1 or later require a Grafana version with this feature. These include:
```

**Review by chri2547 — COMMENTED** [review]
A few suggestions. Thanks for the contribution!

**Comment by chri2547** [comment]
> Not sure about backporting, feels like we might need to but I'm not sure if we should do every version or not. @joanlopez do you know how we have been treating these kinds of docs pr:s recently?

Let's backport to v9.3. I'll add that label.

**Comment by xlson** [comment]
Thanks for the feedback @chri2547! @joanlopez will take over from here. We discussed the pr a bit in a 1-1 and came to the conclusion that it probably would be nice towards the users to backport it to all the old versions where it's supported, but feel free to change that if you disagree.

**Comment by chri2547** [comment]
> Thanks for the feedback @chri2547! @joanlopez will take over from here. We discussed the pr a bit in a 1-1 and came to the conclusion that it probably would be nice towards the users to backport it to all the old versions where it's supported, but feel free to change that if you disagree.

Great idea, works for me!

**Comment by joanlopez** [comment]
Hey @chri2547, I approved all your suggestions and it lgtm, do you think it's fine to merge it? 🤔  Thanks!

File: docs/sources/setup-grafana/configure-grafana/_index.md [hunk 5]
```diff
@@ -1821,6 +1821,12 @@ This option does not require any configuration.
 
 Options to configure a remote HTTP image rendering service, e.g. using https://github.com/grafana/grafana-image-renderer.
 
+#### renderer_token
+
+> **Note**: Available in Grafana v9.1.2 and Image Renderer v3.6.1 or later versions.
```
Comment (chri2547) [inline docs/sources/setup-grafana/configure-grafana/_index.md:1826]:
```suggestion
> **Note**: Available in Grafana v9.1.2 and Image Renderer v3.6.1 or later.
```

File: docs/sources/setup-grafana/image-rendering/_index.md [hunk 6]
```diff
@@ -54,6 +54,33 @@ docker run -d --name=renderer --network=host -v /some/path/config.json:/usr/src/
 
 You can see a docker-compose example using a custom configuration file [here](https://github.com/grafana/grafana-image-renderer/tree/master/devenv/docker/custom-config).
 
+### Security
+
+> **Note:** This feature is available in Image Renderer v3.6.1 and later.
+
+You can restrict access to the rendering endpoint by specifying a secret token. The token should be configured in the Grafana configuration file and the renderer configuration file. This token is important when you run the plugin in remote rendering mode. 
+
+Renderer version v3.6.1 or later require a Grafana version with this feature. These include:
```
Comment (chri2547) [inline docs/sources/setup-grafana/image-rendering/_index.md:63]:
```suggestion
Renderer versions v3.6.1 or later require a Grafana version with this feature. These include:
```

**Review by chri2547 — APPROVED** [review]

**Comment by joanlopez** [comment]
Thanks @chri2547, really appreciated! 💟

**Pull request merged**